kapitan/utils.py

```diff
@@ -309,18 +311,22 @@ def get_directory_hash(directory):
                     with open(file_path, 'r') as f:
                         hash.update(sha256(f.read().encode("UTF-8")).hexdigest().encode("UTF-8"))
```

Comment:
why do you need to `encode("UTF-8"` to generate the `directory_hash` ?

If you have any binary files in the code base then this will break.

Reply:
`sha256()` doesn't like it if we don't encode the result of read. I'll update the code to support binary files.

Reply:
Just tested, binary files should be supported now,